docs/observe.md

```diff
@@ -0,0 +1,131 @@
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Usage](#usage)
+- [Customization](#customization)
+- [API](#api)
+- [Example](#example)
+- [Implementation Details](#implementation-details)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+</details>
+
+## Usage
+
+```js
+import { observe } from 'twind/observe'
+
+observe(document.body)
+
+document.body.innerHTML = `
+  <main class="h-screen bg-purple-400 flex items-center justify-center">
+    <h1 class="font-bold text(center 5xl white sm:gray-800 md:pink-700)">
+      This is Twind!
+    </h1>
+  </main>
+`
+```
+
+> [live and interactive shim demo](https://esm.codes/#aW1wb3J0IHsgb2JzZXJ2ZSB9IGZyb20gJ3R3aW5kL29ic2VydmUnCgpvYnNlcnZlKGRvY3VtZW50LmJvZHkpCgpkb2N1bWVudC5ib2R5LmlubmVySFRNTCA9IGAKICA8bWFpbiBjbGFzcz0iaC1zY3JlZW4gYmctcHVycGxlLTQwMCBmbGV4IGl0ZW1zLWNlbnRlciBqdXN0aWZ5LWNlbnRlciI+CiAgICA8aDEgY2xhc3M9ImZvbnQtYm9sZCB0ZXh0KGNlbnRlciA1eGwgd2hpdGUgc206Z3JheS04MDAgbWQ6cGluay03MDApIj4KICAgICAgVGhpcyBpcyBUd2luZCEKICAgIDwvaDE+CiAgPC9tYWluPgpg)
+
+All twind syntax features like [grouping](./grouping.md) are supported.
+
+> If you want to simplify the instantiation and automatically observe take look at [twind/shim](./installation.md#twindshim).
+
+## Customization
+
+`twind/observe` uses the default/global `tw` instance if not configured otherwise. You can provide a custom instance in several ways:
+
+```js
+import { create } from 'twind'
+import { observe, createObserver } from 'twind/observe'
+
+// Create a custom instance
+const instance = create(/* ... */)
+
+// 1. As second parameter
+observe(document.body, instance)
+
+// 2. As this context
+observe.call(document.body)
+observe.bind(instance)(document.body)
+
+// 3. Use the factory
+createObserver(instance).observe(document.body)
+```
+
+## API
+
+```js
+import { createObserver, observe } from 'twind/observe'
+
+const observer = createObserver(/* custom instance */)
+// Or to start observing an element right away
+// const observer = observe(document.body)
+
+// Start observing a node
+observer.observe(node)
+
+// Stop observing
+observer.disconnect(node)
+```
+
+## Example
+
+> This example show how a custom observer instance can be used to shim a web component.
+
+```js
+import { LitElement, html } from 'lit-element'
+import { create, cssomSheet } from 'twind'
+import { createObserver } from 'twind/observe'
+
+// 1. Create separate CSSStyleSheet
+const sheet = cssomSheet({ target: new CSSStyleSheet() })
+
+// 2. Use that to create an own twind instance
+const instance = create({ sheet })
+
+class TwindElement extends LitElement {
+  // 3. Apply the same style to each instance of this component
+  static styles = [sheet.target]
+
+  // 4. Start observing class attributes changes
+  connectedCallback() {
+    this._observer = createObserver(instance).observe(this.renderRoot)
```

Comment:
lit has a connectedCallback you need to call, so should really `super.connectedCallback()` here. same for the `disconnectedCallback` below

Reply:
Updated the examples.